requirements/base.txt

```diff
@@ -10,3 +10,4 @@ scipy>=1.8
 bayesian-optimization>=1.2.0
 tensorboard>=2.11.0
 multiprocess
+pynvml
```

Comment:
How about making it more specific by adding the version? and I think `requirements/dev.txt` is more proper location.

Reply:
IHMO, it seems that using more robust version is better to avoid a conflict with other packages. But it's better to set major version. I'll update it.
And in terms of `dev.txt`, because it's `otx train` argument, it should be placed in `base.txt`. If not, it raises an error in case where user tries to use this feature after installing otx using 'pip install otx[classification]`.

Reply:
version is updated at [22ad518](https://github.com/openvinotoolkit/training_extensions/pull/2500/commits/22ad518e43478297aafd7a5c9cf47d7adf1593c4)